Dalamud/Utility/Util.cs

```diff
@@ -672,6 +675,40 @@ internal static string GetRandomName()
         return names.ElementAt(rng.Next(0, names.Count() - 1)).Singular.RawString;
     }
 
+    /// <summary>
+    /// Print formatted GameObject Information to ImGui
+    /// </summary>
+    /// <param name="actor">Game Object to Display.</param>
+    /// <param name="tag">Display Tag.</param>
+    /// <param name="resolveGameData">If the GameObjects data should be resolved.</param>
+    internal static void PrintGameObject(GameObject actor, string tag, bool resolveGameData)
```

Comment:
non-actionable note/random thought: imo this is probably better in `ImGuiHelpers`, but I'd leave it here for now because there's several methods like this here already. ideally we should clean up this `Util` class because it's a bit overloaded, but because it exposes other public apis, I don't think we can do much without a major version/api bump

Reply:
I do think this particular draw is pretty specific to the data window, which is why I marked it internal instead of public.
Personally I wouldn't draw gameobject data like this for my own debugging, I would want to draw data more specific to what I cared about.